src/nodes/tab.gd

```diff
@@ -18,3 +19,6 @@ func _ready():
 	
 func set_texture(texture: Texture2D):
 	$EnemySprite2D.texture = texture
+
+func got_attacked():
```

Comment:
just missing which tab got it and then calling the `close` method
after that we need to emit the `player_points_updated` signal